controller/src/beerocks/bml/bml_defs.h

```diff
@@ -599,7 +599,7 @@ struct BML_CLIENT_CONFIG {
 
 struct BML_CLIENT {
     // Client MAC.
-    char sta_mac[BML_MAC_ADDR_LEN];
+    uint8_t sta_mac[BML_MAC_ADDR_LEN];
```

Comment:
commit **title** should be:
`bml: BML_CLIENT: change sta_mac type to unit8_t`
or
`bml: change sta_mac type in BML_CLIENT to unit8_t`

1. bml not bcl
2. current title doesn't say anything or doesn't say enough

Reply:
Changed